powerline/renderers/lemonbar.py

```diff
@@ -36,10 +37,26 @@ def hl(self, contents, fg=None, bg=None, attrs=None):
 		return text + contents + '%{F-B--u}'
 
 	def render(self, *args, **kwargs):
+		self.segment_info['output'] = kwargs.get('matcher_info')
```

Comment:
ah, you are right. I'm not using it right now, but I also don't know if it may be useful at times. Do you think I should include it or not?